ES/ESum042.xml

```diff
@@ -588,31 +588,31 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                          
                          <msItem xml:id="p3_i1.1">
                             <locus from="139ra" to="139vb"/>
-                            <title type="complete" ref="LIT4669TaammeraQi">Miracle 1. About a pious king who accept to be a priest in the land of Roḥa</title>
+                            <title type="complete" ref="LIT5732MiracleCyriacus">Miracle 1. About a pious king who accept to be a priest in the land of Roḥa</title>
                             <textLang mainLang="gez"/>   
                             <incipit xml:lang="gez">
                                 በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡ ነዋ፡ ጸሐፍነተ፡ አምሪሁ፡ ለቂርቆስ፡ ሰማዕት፡ 
                                 ንስቲተ፡ እምብዙኃ፡ ከመ፡ ኢይኩን፡ ዝንጋዔ፡ ለሰማዕያን፡ ጸሎቱ፡
-                                ወበረከቱ፡ የሀሉ፡ ምስለ፡ ክፍለ፡ ማርያም፡ ለዓለመ፡ ዓለም፡ አሜን። ወሀሎ፡ ፩ንጉሥ፡ ጻድቅ፡ ዘተወክፈ፡ ሢመተ፡ ክህነት፡ እምአፈ፡ 
+                                ወበረከቱ፡ የሀሉ፡ ምስለ፡ <persName ref="PRS13964KeflaMaryam">ክፍለ፡ ማርያም፡ </persName>ለዓለመ፡ ዓለም፡ አሜን። ወሀሎ፡ ፩ንጉሥ፡ ጻድቅ፡ ዘተወክፈ፡ ሢመተ፡ ክህነት፡ እምአፈ፡ 
                                አምላኩ፡ በሀገረ፡ ሮሐ፡ ቅድስት፡ 
                             </incipit>
                       </msItem>
                          
                          <msItem xml:id="p3_i1.2">
                             <locus from="140ra" to="140rb"/>
-                            <title type="complete" ref="LIT4669TaammeraQi">Miracle 2. About a believer in a church dedicated to Qirqos</title>
+                            <title type="complete" ref="LIT5720MiracleCyriacus" cert="medium">Miracle 2. About a believer in a church dedicated to Qirqos</title>
```

Comment:
sure? we also have
[Miracle of St Cyricus (Qirqos): St Cyricus and the guardian of the church of St Cyriacus, in the region of Māy Ḍaʿādā](https://betamasaheft.eu/LIT5743MiracleCyriacus)
[Miracle of St Cyricus (Qirqos): St Cyricus and the son of a presbyter of the church devoted to him](https://betamasaheft.eu/LIT5734MiracleCyriacus)
[Miracle of St Cyricus (Qirqos): St Cyricus, learned priests of his church and a hermit](https://betamasaheft.eu/LIT5748MiracleCyriacus)

Reply:
Not sure. The miracles are not very easy to identify, due to various reasons and esp. short quotations. I would leave UM 042 as it is, for the moment, to come back to the issue a bit later.